docs/tutorials/google/calibration_api.ipynb

```diff
@@ -0,0 +1,609 @@
+{
```

Comment:
<p>kindof confusing that theta and phi are set to "False" below if we say we can get estimates for all angles except chi.</p><p> </p><p>It could be mildly helpful to put the latin-alphabet-spelling (chi) after the symbol so people can relate it to the code. It looks like an X</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Set all to <code>False</code>  execpt for <code>chi</code>,  + added the latin spelling of chi in the text.</p>